apps/desktop/desktop/src/main/java/bisq/desktop/main/content/bisq_easy/offerbook/BisqEasyOfferbookController.java

```diff
@@ -256,10 +257,19 @@ protected void selectedChannelChanged(ChatChannel<? extends ChatMessage> chatCha
                 BisqEasyOfferbookChannel channel = (BisqEasyOfferbookChannel) chatChannel;
 
                 // FIXME (low prio): marketChannelItems needs to be a hashmap
-                model.getMarketChannelItems().stream()
+                Optional<MarketChannelItem> marketChannelItem = model.getMarketChannelItems()
+                        .stream()
                         .filter(item -> item.getChannel().equals(channel))
-                        .findAny()
-                        .ifPresent(item -> model.getSelectedMarketChannelItem().set(item));
+                        .findAny();
+
+                marketChannelItem.ifPresent(item -> {
+                        model.getSelectedMarketChannelItem().set(item);
+                        model.getShowsExpiredMessagesIndicator().set(!item.getChannel()
+                                .getChatMessages()
+                                .stream()
+                                .findFirst()
+                                .isPresent());
+                });
```

Comment:
Why keep a local marketChannelItem variable here?

That code:
```
!item.getChannel()
                                .getChatMessages()
                                .stream()
                                .findFirst()
                                .isPresent()
```
could be replaced by `!item.getChannel().getChatMessages().isEmpty()`

Reply:
Sorry for that.